<launch>
  <!-- Arguments -->
  <arg name="configuration_basename" default="turtlebot3_lds_2d.lua"/>
  <arg name="scan_topic" default="/scan_filtered"/>

  <!-- frontier_exploration -->
  <node pkg="explore_lite" type="explore" respawn="false" name="explore" output="screen">
    <remap from="/scan" to="$(arg scan_topic)"/>
    <param name="robot_base_frame" value="base_link"/>
    <!-- <param name="costmap_topic" value="move_base/global_costmap/costmap"/>
    <param name="costmap_updates_topic" value="move_base/global_costmap/costmap_updates"/> !-->
     <param name="costmap_topic" value="/map"/>
    <param name="costmap_updates_topic" value="/map_updates"/> 
    <param name="visualize" value="true"/>
    <param name="planner_frequency" value="0.6"/>
    <param name="progress_timeout" value="60.0"/>
    <param name="potential_scale" value="10.0"/>
    <param name="orientation_scale" value="0.0"/>
    <param name="gain_scale" value="1.0"/>
    <param name="transform_tolerance" value="0.3"/>
    <param name="min_frontier_size" value="0.5"/>
  </node>
</launch>

